helper/include/odp/helper/cli.h

```diff
@@ -54,20 +59,55 @@ typedef struct {
  */
 void odph_cli_param_init(odph_cli_param_t *param);
 
+/**
+ * Initialize CLI helper
+ *
+ * This function initializes the CLI helper. It must be called before
+ * odph_cli_register_command() and odph_cli_start().
+ *
+ * @param instance ODP instance
+ * @param param CLI server parameters to use
+ * @retval 0 Success
+ * @retval <0 Failure
+ */
+int odph_cli_init(const odp_instance_t instance, const odph_cli_param_t *param);
+
+/**
+ * Register a user defined command
+ *
+ * Register a command with a name, function, and an optional help text. The
+ * registered command is displayed in the output of the "help" command. When the
+ * command is invoked by the CLI client, the registered function is called with
+ * the parameters supplied by the CLI client.
+ *
+ * Command names are case-insensitive. In the CLI client, they are displayed in
+ * the case they were registered in, but they may be invoked using any case. No
+ * two commands may have the same name, in terms of case-insensitive string
+ * comparison.
+ *
+ * This function returns with a failure if called while the CLI server is
+ * running (after odph_cli_start()).
+ *
+ * @param name Command name (case-insensitive)
+ * @param func Command function
+ * @param help Help or description for the command. This appears in the output
+ *             of the "help" command. May be NULL.
+ * @retval 0 Success
+ * @retval <0 Failure
+ */
+int odph_cli_register_command(const char *name, odph_cli_user_cmd_func_t func, const char *help);
+
 /**
  * Start CLI server
  *
  * Upon successful return from this function, the CLI server will be
  * accepting client connections. This function spawns a new thread of
  * type ODP_THREAD_CONTROL using odp_cpumask_default_control().
  *
- * @param instance ODP instance
- * @param param CLI server parameters to use
  * @retval 0 Success
  * @retval <0 Failure
  */
-int odph_cli_start(const odp_instance_t instance,
-		   const odph_cli_param_t *param);
+int odph_cli_start();
```

Comment:
Prototype needs to be int odph_cli_start(void)

Documentation could repeat shortly here the correct start up order: odph_cli_init() -> optionally add commands -> start()

Reply:
Fixed in v2.